tests/test_config_noise.py

```diff
@@ -776,12 +776,13 @@ def test_whiten():
     np.testing.assert_almost_equal(im4b.array, im4a.array, decimal=5)
 
     # Repeat with gain != 1
-    config['image']['noise']['gain'] = 3.7
+    gain = 3.7
+    config['image']['noise']['gain'] = gain
     galsim.config.RemoveCurrent(config)
     im5a = im1a.copy()
-    rn = math.sqrt(25-cv1a * 3.7**2)
+    rn = math.sqrt((25/gain-cv1a) * gain**2)
```

Comment:
I don't think this edit is right.  It's still just 25, which is the nominal read_noise variance in e-.  You want to subtract off the current variance in e-, which is cv1a * gain**2.  Then sqrt to get read_noise sigma, still in e-.

Reply:
Hmmmm. The config docs say read noise should be in ADU, not e-.

```
read_noise = float_value (default = 0.0) The CCD read noise in ADU.
```

I guess the class expects different and a conversion is done elsewhere?